events_frontend/src/Main.js

```diff
@@ -132,21 +132,21 @@ class Main extends Component {
             <Redirect to={"/"} />
           </Switch>
         ) : (
-          <Switch>
-            {Object.values(routes.noAuth).map(obj => (
-              <Route
-                exact
-                key={obj.route}
-                path={obj.route}
-                component={obj.component}
-              />
-            ))}
-            {Object.values(routes.auth).map(obj => (
-              <Redirect key={obj.route} from={obj.route} to={"/login/"} />
-            ))}
-            <Redirect to={"/"} />
-          </Switch>
-        )}
+            <Switch>
+              {Object.values(routes.noAuth).map(obj => (
+                <Route
+                  exact
+                  key={obj.route}
+                  path={obj.route}
+                  component={obj.component}
+                />
+              ))}
+              {Object.values(routes.auth).map(obj => (
+                <Redirect key={obj.route} from={obj.route} to={"/login/"} />
+              ))}
+              <Redirect to={"/"} />
+            </Switch>
+          )}
```

Comment:
You should avoid this indentation only change. It is very likely to cause merge conflicts, and your change is not consistent with the rest of the code style